install.js

```diff
@@ -24,6 +29,8 @@ var app = builder();
 var masterTenant = false;
 var superUser = false;
 
+var isVagrantInstallation = false;
```

Comment:
this var could be initialized here with the function suggested in my comment below.